### DISCUSSION TITLE
How do we determine which font being used in the drawOptions

### DISCUSSION THREAD

**Discussion by gayathrijosyula** [BinaryKits/ZPLUtility#175] [General]
How can we determine which font is being used in the drawOptions? Additionally, how can we identify whether it is using Font 0 or Font 1 from drawOptions?

https://github.com/BinaryKits/BinaryKits.Zpl#font-mapping:~:text=using%20SkiaSharp%3B,)%3B%0A%7D

**Comment by primo-ppcg** [comment]
The font used is specified in the ZPL, using the [`^A` command](https://docs.zebra.com/content/tcm/us/en/printers/software/zebra-zpl-ii,-zbi-2,-set-get-do,-mirror,-wml-programming-guide/c-zpl-zpl-commands/r-zpl-a.html) for a single field, or [`^CF` command](https://docs.zebra.com/content/tcm/us/en/printers/software/zebra-zpl-ii,-zbi-2,-set-get-do,-mirror,-wml-programming-guide/c-zpl-zpl-commands/r-zpl-cf.html) to change the default. The default is initially font `0` for text fields. Barcode interpretation lines are rendered with font `A`.
